frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -89,3 +91,29 @@ export const alphanumericCompare = (a: string, b: string): number => {
     sensitivity: 'base',
   });
 };
+
+export const translationForResourceKind = {
+  // t('console-shared~Helm Release')
+  HelmRelease: `console-shared~Helm Release`,
+};
+
+export const labelKeyForNodeKind = (kindString: string) => {
+  const model: K8sKind | undefined = modelFor(kindString);
+  if (model) {
+    if (model.labelKey) {
+      return model.labelKey;
+    }
+    return model.label;
+  }
+  if (translationForResourceKind[kindString]) {
+    return translationForResourceKind[kindString];
+  }
+  return _.startCase(kindString);
+};
+
+type PageTitleContextType = {
```

Comment:
Export this type so that you can use it:

```suggestion
export type PageTitleContextValues = {
```